src/RamTranslator.cpp

```diff
@@ -365,6 +366,9 @@ std::unique_ptr<RamValue> translateValue(const AstArgument* arg, const ValueInde
         // here we look up the location the aggregation result gets bound
         auto loc = index.getAggregatorLocation(*agg);
         val = std::unique_ptr<RamValue>(new RamElementAccess(loc.level, loc.component, loc.name));
+    } else if (const AstSubroutineArgument* subArg = dynamic_cast<const AstSubroutineArgument*>(arg)) {
+        // TODO: RamNumber(0) temporary
```

Comment:
If you still need this TODO then preferable make it
```
// TODO(taipan-snake): ...
```
or make an issue and reference it.
```
// TODO(#000): ...
```